### PR TITLE
Fixes detection multitool

### PR DESCRIPTION
**Pull request by silicons**
Woops I accidently broke it by using range with (src) instead of get_turf(src)
Honk
Sorry.

### CODE REVIEW

File: code/game/objects/items/devices/multitool.dm [hunk 1]
```diff
@@ -76,7 +75,8 @@
 	track_delay = 5
 
 /obj/item/device/multitool/ai_detect/admin/multitool_detect()
-	for(var/mob/J in range(rangewarning,src))
+	var/turf/our_turf = get_turf(src)
+	for(var/mob/J in range(rangewarning,get_turf(src)))
```
Comment (RemieRichards) [inline code/game/objects/items/devices/multitool.dm]: Cmon. I'll give you one guess.

Comment (kevinz000) [inline code/game/objects/items/devices/multitool.dm]: Hmm?

Comment (RemieRichards) [inline code/game/objects/items/devices/multitool.dm]: You're doing something wrong~, 1 guess.

Comment (kevinz000) [inline code/game/objects/items/devices/multitool.dm]: wait uhhh declaring new vars in the detection and not in the multitool? :p 
sorry still very very new and reading already coded things to figure out how to make things

Comment (RemieRichards) [inline code/game/objects/items/devices/multitool.dm]: You're doing ```get_turf(src)``` twice, just use ```our_turf``` here.

Comment (kevinz000) [inline code/game/objects/items/devices/multitool.dm]: OH
WOOPS
rofl didnt see that hold on

File: code/game/objects/items/devices/multitool.dm [hunk 2]
```diff
@@ -76,7 +75,8 @@
 	track_delay = 5
 
 /obj/item/device/multitool/ai_detect/admin/multitool_detect()
-	for(var/mob/J in range(rangewarning,src))
+	var/turf/our_turf = get_turf(src)
+	for(var/mob/J in range(rangewarning,our_turf))
```
Comment (xxalpha) [inline code/game/objects/items/devices/multitool.dm]: Use ``urange()`` for ranges of 9 or higher, it's faster and it already does the get_turf() for you.

**Pull request merged**